ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/correction/ExtraneousLinesResolution.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ *  Copyright (c) 2023 IBM Corporation and others.
+ *
+ *  This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License 2.0
+ *  which accompanies this distribution, and is available at
+ *  https://www.eclipse.org/legal/epl-2.0/
+ *
+ *  SPDX-License-Identifier: EPL-2.0
+ *
+ *  Contributors:
+ *     IBM Corporation - initial API and implementation
+ *     Gireesh Punathil <gpunathi@in.ibm.com> Initial implementation
+ *******************************************************************************/
+
+package org.eclipse.pde.internal.ui.correction;
+
+import org.eclipse.core.resources.IMarker;
+import org.eclipse.jface.text.IDocument;
+import org.eclipse.jface.text.IRegion;
+import org.eclipse.pde.internal.core.text.bundle.BundleModel;
+import org.eclipse.pde.internal.ui.PDEUIMessages;
+
+/**
+ * <p>Represents a resolution to the problem of the bundle manifest
+ * having extraneous lines.</p>
+ */
+public class ExtraneousLinesResolution extends AbstractManifestMarkerResolution {
+
+	/**
+	 * Creates a new resolution
+	 *
+	 * @param type
+	 *            {@link AbstractPDEMarkerResolution#REMOVE_TYPE} to delete
+	 *            lines
+	 */
+	public ExtraneousLinesResolution(int type, IMarker marker) {
+		super(type, marker);
+	}
+
+	/**
+	 * Resolves the problem by extracting the empty line and removing it.
+	 */
+	@Override
+	protected void createChange(BundleModel model) {
```

Comment:
Please check how **multiple** empty lines are handled via quick fix.
After applying quickfix the manifest is corrupted, some **proper** lines are removed:

![image](https://user-images.githubusercontent.com/964108/233020255-9f8171e0-a7e0-4fbb-87cb-6e58137c9c10.png)
After 
![image](https://user-images.githubusercontent.com/964108/233020409-9a41a083-e1c8-4e98-ac9d-e9486221c944.png)

Reply:
yes, this is solved now.